src/foam/nanos/theme/Themes.js

```diff
@@ -177,15 +177,17 @@ Later themes:
       if ( user != null ) {
         DAO groupDAO = (DAO) x.get("groupDAO");
         Group group = user.findGroup(x);
-        while ( group != null ) {
-          Theme groupTheme = group.findTheme(x);
+        Theme groupTheme = null;
+        while ( groupTheme == null &&  group != null) {
+          groupTheme = group.findTheme(x);
           if ( groupTheme != null ) {
             theme = (Theme) theme.fclone().copyFrom(groupTheme);
             if ( ! SafetyUtil.isEmpty(group.getDefaultMenu()) ) {
               theme.setDefaultMenu(group.getDefaultMenu());
             }
+          } else {
+            group = (Group) groupDAO.find(group.getParent());
```

Comment:
can we just break at after applying groupTheme instead?

Reply:
like this ?
```java
          if ( groupTheme != null ) {
            theme = (Theme) theme.fclone().copyFrom(groupTheme);
            if ( ! SafetyUtil.isEmpty(group.getDefaultMenu()) ) {
              theme.setDefaultMenu(group.getDefaultMenu());
            }
          break;
}
```

Reply:
yes, just break at the end of the if ( groupTheme != null ) { block }

Reply:
So this should be a one line delta. 
All you need is 'break;' 
You can remove everything else you added.